src/main/proto/spine/options.proto

```diff
@@ -159,7 +201,8 @@ extend google.protobuf.FieldOptions {
     // Once set, the `id` field cannot be changed.
     //
     // Use `(if_set_again).error_msg` option to specify a custom error message that will be shown
```

Comment:
I would change this "shown" to something that describes the real world better. Maybe it could be as follows:

"... that will be used for composing the error upon attempting to re-assign the field value."